### DISCUSSION TITLE
Mini-Logo img tags

### DISCUSSION THREAD

**Discussion by mat-ti** [kimai/kimai#2203] [General]
At the Admin Settings for mini logo, write down the url isnt enough, i had to add the img tags like
`<img src="URL">`. Is this a feature or a bug?

**Comment by kevinpapst** [comment]
The "Mini Logo" should contain text, but the field allows to enter HTML tags as well (which wasn't intended, so it is a bug, but lets call it a feature).

See https://www.kimai.org/documentation/branding.html

**Comment by BeckeBauer** [comment]
I added my logo... So please keep the option to use also pics instead of letters...

**Comment by BeckeBauer** [comment]
in the demo, I can see the logo to be used but no hint as to how to do it 
<img width="967" alt="Bildschirmfoto 2023-05-21 um 22 13 25" src="https://github.com/kimai/kimai/assets/51003649/ccf9a670-b277-4a23-950e-b157069e285b">
the code in the field company name used in V1 is no longer working - any advice?

**Reply by kevinpapst** [reply]
Not supported anymore. But I am certain, that you can create some custom CSS for your logo there.

**Comment by kevinpapst** [comment]
The initial question was for Kimai 1.???
This is not maintained anymore, so the question can be closed.
